### PR TITLE
Fix node services title and cards

### LINKED ISSUE #141: Services title  font needs to be checked in mobile

### LINKED ISSUE #142: Services card spacing needs to be checked in mobile

### PR DESCRIPTION
**Pull request by alexandercyu**
Fixes #141 
Fixes #142 
- Fix node services title font and spacing
- Fix node services cards alignment

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
This changes the spacing of these cards when the screen width is `> sm`. Specifically, buttons are too close together: 

image

File: static/css/marble.css [hunk 1]
```diff
@@ -1096,6 +1096,50 @@ h6{
...
+        margin-top: 0px;
+        margin-right: 0px;
+        margin-bottom: 0px;
+        padding-left: 15px;
+        padding-right: 15px;
+        padding-top: 15px;
+        padding-bottom: 15px;
+    }
+
+    .margin-node-card-title{
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/marble.css:1127]: The widths of the cards are different at different screen sizes.  sm is 345px, lg is 322px.  The paddings are different at different screen sizes as well.

[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/marble.css:1127]: Moved out of media queries.

**Comment by alexandercyu** [comment]
> This changes the spacing of these cards when the screen width is `> sm`. Specifically, buttons are too close together:
> 
> image

Fixed the space between buttons for larger screen sizes.

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
On laptop browser, the services are divided in to 2 columns. where as the need to be presented in 3X3. Able to see that grip on desktop but since most of our user user a laptop to browse the site we should;d focus on creating 3X3 grid presentation for the service on Laptop browser.
Screenshot 2024-05-30 at 10 46 52 AM

Please check the gap between the Service heading, body and buttons, needs to be 10px gap on browser site.
Screenshot 2024-05-30 at 10 53 18 AM

Mobile version service cards looks good!

**Comment by mishaschwartz** [comment]
@alexandercyu Here's an example of how to implement this is a dynamic grid which might get us closer to the layout that we want: https://stackoverflow.com/a/74656166/5992438

**Comment by alexandercyu** [comment]
> On laptop browser, the services are divided in to 2 columns. Whereas, they need to be presented in 3X3 grid. Able to see that on desktop monitor(large screen) but since most of our user use a laptop to browse the site we should focus on creating 3X3 grid presentation for the services on Laptop browser as well. Screenshot 2024-05-30 at 10 46 52 AM
> 
> Please check the gap between the Service heading, body and buttons, needs to be 10px gap on browser site. Screenshot 2024-05-30 at 10 53 18 AM
> 
> Mobile version service cards looks good!

Adjusted the spacing so it will be three per row in laptop screens.

Changed space above buttons to be 10px.

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
Grid is fixed, looks good 
Still don't see the 10px gap between buttons
Screenshot 2024-05-30 at 11 34 54 AM

**Comment by alexandercyu** [comment]
> Grid is fixed, looks good Still don't see the 10px gap between body and buttons Screenshot 2024-05-30 at 11 34 54 AM

Screenshot 2024-05-30 at 11 46 01 AM (2)


Should be there now.

**Review by mishaschwartz — COMMENTED** [review]
The `p` element above the buttons has a bottom margin of 16px which is added to the 10px top margin from the buttons to increase the gap to 26px

**Comment by alexandercyu** [comment]
> The `p` element above the buttons has a bottom margin of 16px which is added to the 10px top margin from the buttons to increase the gap to 26px

Removed the default margin from the p tag.

**Review by ShrutiK100 — APPROVED** [review]
Looks good!

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**